Cookbook.md

```diff
@@ -1,13 +1,13 @@
 # Cookbook
 
 This document describes how I update the list, just so you know how it works if
-I get hit by a truck. The near-future goal is to automate this process.
+I get hit by a truck.
 
-1. Run `brew update` and check if there are new formulae
-2. If so, install them. You can also install updated formulae, they might have
-   changed their binaries.
-3. In the root directory of this tap, run `brew which-update --commit`
-4. Push.
+1. Run `brew update`
+2. Run `brew which-update --commit`. It’ll update installed formulae and fetch
```

Comment:
How long does this take? Might be worth investigating if it can be done in e.g. Travis CI?

Reply:
It depends on how many new (bottled) formulae there are since the last update and how big they are, so I’d say 3-4 minutes max.

Reply:
@bfontaine May be worth trying to add this to the homebrew-core Travis CI job?